### ISSUE TITLE
Allow several licenses to be specified

### ISSUE THREAD

**Issue by micha137** [Memnarch/Delphinus#30]
Maybe this is even possible, but not documented in https://github.com/Memnarch/Delphinus/wiki/Delphinus.Info.json.  My use case is https://github.com/graphics32/graphics32/blob/master/Source/License.txt, which states MPL-1.1 and LGPL-1.1,

**Comment by Memnarch** [comment]
Thanks for the tip. Currently it is just a textfield. Use a semicolon to seperate them. Write them right next to each other, as you do with platforms, like:

MPL-1.1;LGPL-1.1

Currently, it'll just display this text, but the seperato will allow me to process it, if i implement something.

**Comment by rlebeau** [comment]
Yes, please.  I'm thinking of adding [Indy](http://www.indyproject.org) to Delphinus, but it uses a [dual license](http://www.indyproject.org/License/index.aspx) - a **modified** BSD license (how would I specify that in the Delphinus.Info.json file?), in addition to MPL v1.1.

**Comment by Memnarch** [comment]
Will prioritize this to get you started :)

**Comment by Memnarch** [comment]
did some quick changes in a branch that should do(will use the suggestion with the license array now). Commandline has been extended to support displaying a specific license of a given package, too.
However it's quite late so i'll merge tomorrow.
